src/_tests_/Profile.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import Profile from '../components/Profile/Profile';
+import store from '../redux/store';
+
+describe('Test Profile components', () => {
+  it('renders correctly', () => {
+    const tree = renderer
+      .create(
+        <Provider store={store}>
+          <Profile />
+        </Provider>,
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] The test for the file failed. Kindly work on it so all tests cases passed successfully 
<img width="591" alt="Screenshot 2023-04-14 at 2 30 55 PM" src="https://user-images.githubusercontent.com/50941074/232074380-80808e4d-b6bb-4fa5-96d3-0b792823414b.png">